openid-caep-interoperability-profile-1_0.md

```diff
@@ -93,6 +98,14 @@ Credential Change
 # Common Requirements {#common-requirements}
 The following requirements are common across all use-cases defined in this document.
 
+## Network layer protection
+* The SSF transmitter MUST offer TLS protected endpoints and shall establish connections to other servers using TLS. TLS connections MUST be set up to use TLS version 1.2 or later.
+* When using TLS 1.2, follow the recommendations for Secure Use of Transport Layer Security in [RFC7525]{{RFC7525}}.
+* The SSF receiver MUST perform a TLS server certificate signature checks, chain of trust validations, expiry and revocation status checks before calling the SSF transmitter APIs, as per [RFC6125]{{RFC6125}}.
+
+## CAEP specification version
+This specification supports CAEP {{CAEP}} events from Implementer's Draft 1
```

Comment:
Why wouldn't we use [CAEP draft 2](https://openid.net/specs/openid-caep-specification-1_0-02.html)?

Reply:
+1. We should make this for CAEP implementer's draft 2